pkg/secretless/plugin/connector/resources_test.go

```diff
@@ -0,0 +1,22 @@
+package connector
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/cyberark/secretless-broker/internal/pkg/log"
+)
+
+func TestNewReourcesWrapper(t *testing.T) {
```

Comment:
typo.  also, i'd remove `Wrapper`.  It's a test of `NewResources`, so `TestNewResources` is exactly right.  You can to the test suite subtests with `t.Run` per normal if you need to add more.